operations/node_management/Add_Remove_Replace_NCNs/Add_NCN_Data.md

```diff
@@ -535,6 +536,97 @@ The NCN MAC addresses need to be collected using the [Collect NCN MAC Addresses]
     Class = "River"
     ```
 
+### IPv6
+
+This step is only necessary for systems with IPv6 enabled on their CMN and CHN networks.
+
+#### Determine if IPv6 is enabled
+
+1. To preview whether IPv6 has been setup on the current system, dump the `CIDR6` values for the CHN and CMN.
```

Comment:
Need command prompts for the new steps being added

Reply:
Thank you, I originally left them since I was already touching a lot, but I won't say no if someone else wanted to fix those indentations.

I'll fix the prompts.